site/_data/mr-menus.yml

```diff
@@ -0,0 +1,120 @@
+---
+## MAKE Roanoke Menus
+##
+## Menu Item fields:
+##   name  -- Name to display on Menu Item
+##   path  -- URL of page represented by Menu Item
+##   menu  -- Name of sub-menu
+##
+## Menu fields:
+##   name  -- Name of Menu
+##   list  -- List of Menu Items
+##
+
+##
+## MAKE Roanoke Menus supports three placement menus that can be 
+## displayed on each page: primary, secondary, footer.
+##
+## The primary menu is the menu displayed at the top of each page
+## usually just under the header section or in a side bar.
+##
+## The secondary menu is not yet implemented and its location is TBD.
+##
+## The footer menu is displayed in the footer section of a page.
+##
+## Each of these placement menus are references to other real menus defined 
+## in this YAML page.  They are set to the name of other real menus.  Note,
+## the same real menu can be assigned to one or more of the placement menus.
+##
+primary:    main-menu
+secondary:  
+footer:     main-menu
+
+##
+## The variable menus is an array of real menu names.  Each time a new real
+## menu is defined, its name needs to be added to this list.
+##
+menus:
+  - main-menu
+  - guild-halls
+
+###
+### Real Menu Definitions
+###
+
+## Defining Main Menu
+main-menu:
+  name: "Main Menu"
+  list:
+    - Home:
+      name: "Home"
+      path: "/"
+    - Events:
+      name: "Events"
+      path: "/events"
+    - Makerspace:
+      name: "Makerspace"
+      path: "/makerspace"
+    - AboutUs:
+      name: "About Us"
+      path: "/about"
+
+
+## Example on how to add submenus to menu 
+mock-main-menu:
+  name:  "Main Example Menu"
+  toolTip: "This is the primary menu of the MAKE Roanoke website."
+  list:
+    - Home:
+      name: "Home"
+      path: "/"
+    - Guild Halls:
+      name: "Guild Halls"
+      path: "/guild-halls"
+      menu: "guild-halls"
+    - About Us:
+      name: "About Us"
+      path: "/about"
+    - Makerspace:
+      name: "What is a Makerspace"
+      path: "/makerspace"
```

Comment:
I am confused as to why YML is being compared to JSON.  Liquid, the language that Jekyll uses for programming, is designed to work with YML data, not JSON.

I also do not understand the comment that the About Us menu item will not always be in the same place in the list since the location of the "About Us" menu item in the list does not matter.  The code works for adding a submenu to the "About Us" menu item wherever it is in the menu. (See mr-theme for About Us submenu).